### PR TITLE
docs: Update the README

### PR DESCRIPTION
**Pull request by michaelgrigoryan25**
## Type of change
- [X] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### Development

- [X] Lint rules pass locally
- [X] The code changed/added as part of this pull request has been covered with tests
- [X] All tests related to the changed code pass in development

### Code review

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [x] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [x] Issue from task tracker has a link to this pull request

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,59 +20,74 @@ For a full description of request and response payloads and properties, please s
...
+	client := courier.CourierClient("<YOUR_AUTH_TOKEN>", nil)
+	message := courier.SendMessageRequestBody{
+		"template": "<COURIER_TEMPLATE>",
+			"to": map[string]string{
+				"email": "test@email.com"
+		}
 	}
-        log.Println(requestID)
+
+	id, err := client.SendMessage(context.Background(), message)
```
Comment (vatasha) [inline README.md:50]:
```suggestion
	reqId, err := client.SendMessage(context.Background(), message)
```

File: README.md [hunk 2]
```diff
@@ -20,59 +20,74 @@ For a full description of request and response payloads and properties, please s
...
+func main() {
+	client := courier.CourierClient("<YOUR_AUTH_TOKEN>", nil)
+	message := map[string]interface{}{
+		"template": "<COURIER_TEMPLATE>",
+			"to": map[string]string{
+				"email": "test@email.com"
+			}
 	}
-        log.Println(requestID)
+	id, err := client.SendMessageWithOptions(
```
Comment (vatasha) [inline README.md:80]:
```suggestion
	reqId, err := client.SendMessageWithOptions(
```

**Review by vatasha — COMMENTED** [review]
Minors nits but everything else looks good to me! Thanks for submitting the PR to help us improve our docs 🎉

File: README.md (hunk 2 above)
Comment (michaelgrigoryan25) [reply README.md:80]: I think it should be clear from the code and Courier documentation that sending a message request will return a request ID.

Comment (vatasha) [reply README.md:80]: I agree but ID here feels really vague. I'd rather err on the side of being more verbose so that it leads to less confusion.

Comment (michaelgrigoryan25) [reply README.md:80]: In that case I'd recommend changing it to `reqID` (fully uppercase) at least, to comply with Go naming conventions. See: [Initialisms](https://github.com/golang/go/wiki/CodeReviewComments#initialisms)

Comment (vatasha) [reply README.md:80]: Oh yes, I like that!

**Review by vatasha — APPROVED** [review]
LGTM, thank you! 🎉

**Pull request merged**